packages/dd-trace/src/appsec/telemetry/waf.js

```diff
@@ -98,6 +109,39 @@ function incrementWafRequests (store) {
   }
 }
 
+function incrementTruncatedMetrics (metrics, truncationReason) {
+  const truncationTags = { truncation_reason: truncationReason }
+  appsecMetrics.count('waf.input_truncated', truncationTags).inc(1)
+
+  if (metrics?.maxTruncatedString) {
+    appsecMetrics.distribution('waf.truncated_value_size',
+      { truncation_reason: TRUNCATION_FLAGS.LONG_STRING })
+      .track(metrics.maxTruncatedString)
```

Comment:
these linebreaks and indentations are weeeeeird
either that:
```suggestion
    appsecMetrics.distribution('waf.truncated_value_size', {
      truncation_reason: TRUNCATION_FLAGS.LONG_STRING
    }).track(metrics.maxTruncatedString)
```
or that:
```javascript
    appsecMetrics.distribution(
      'waf.truncated_value_size',
      { truncation_reason: TRUNCATION_FLAGS.LONG_STRING }
    ).track(metrics.maxTruncatedString)
```

Reply:
Done